tests/integration/tests/cluster/test-cluster-overview.spec.js

```diff
@@ -27,6 +27,8 @@ context('Test Cluster Overview', () => {
     cy.contains('Nodes').should('be.visible');
 
     cy.contains('Events').should('be.visible');
+
+    cy.contains('NAT Gateway IP Addresses').should('be.visible');
```

Comment:
Here, probably that needs to be adjusted. You need to apply the config-map (I suggest adding `kyma-info-cm.yaml` with mocked data, and  adjusting the `setup_local_busola` script to  apply this config map during [this step, or add an additional step to apply it)](https://github.com/kyma-project/busola/blob/main/.github/workflows/pull-integration-cluster-k3d.yml#L53)